lib/elixir/lib/process.ex

```diff
@@ -239,10 +239,18 @@ defmodule Process do
   which is not alive or when the given PID exits. Note that timers will not be
   automatically canceled when `dest` is an atom (as the atom resolution is done
   on delivery).
-  """
-  @spec send_after(pid | atom, term, non_neg_integer) :: reference
-  def send_after(dest, msg, time) do
-    :erlang.send_after(time, dest, msg)
+
+  The function accepts one boolean option: `:abs`. When `false` (the default),
+  the `time` is treated as relative to the current monotonic time. When `true`
+  the `time` is an absolute value for the erlang monotonic time when the message
+  should be sent. The concept of the erlang monotonic time (and other time
+  measures kept by the VM) can be explored in the
+  [erlang documentation](http://erlang.org/doc/apps/erts/time_correction.html#Erlang_Monotonic_Time).
+  """
+  @spec send_after(pid | atom, term, non_neg_integer, opts) :: reference when
```

Comment:
Looks like that something that we should fix then.
For this specific case any style will work. :bowtie: